<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>541</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>中通</string>
    </property>
    <widget class="QLineEdit" name="zhongtongFileText">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="zhongtongFileButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>打开文件...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="companyFileButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>70</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>打开文件...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="companyFileText">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="zhongtongCheckButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>开始比对</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>外部：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>内部：</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>541</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>申通</string>
    </property>
    <widget class="QLineEdit" name="shentongFileText">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="shentongFileButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>打开文件...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="companyFileButton_1">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>70</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>打开文件...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="companyFileText_1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="shentongCheckButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>开始比对</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>外部：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>内部：</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Zapfino</family>
      <pointsize>42</pointsize>
     </font>
    </property>
    <property name="text">
     <string>快递账单核对系统</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
